### ISSUE TITLE
tries to install VcXsrv.lnk to a non-existent location

### ISSUE THREAD

**Issue by AlisterH** [aseering/wsl_gui_autoinstall#9]
Sorry, I've lost the actual error message after a problem filing a ticket, but it tries to install a VcXsrv.lnk into C:\Users\alistair\Start Menu\Programs\Startup\, but this location does not exist on my system.  I only have a start menu folder in these places:
````
C:\ProgramData\Microsoft\Windows\Start Menu
C:\Users\alistair\AppData\Roaming\Microsoft\Windows\Start Menu
C:\Users\Default\AppData\Roaming\Microsoft\Windows\Start Menu
C:\Users\User\AppData\Roaming\Microsoft\Windows\Start Menu
C:\Windows\ServiceProfiles\LocalService\AppData\Roaming\Microsoft\Windows\Start Menu
C:\Windows\ServiceProfiles\NetworkService\AppData\Roaming\Microsoft\Windows\Start Menu
C:\Windows\System32\config\systemprofile\AppData\Roaming\Microsoft\Windows\Start Menu
````

**Comment by aseering** [comment]
Hm, interesting.  The script tries to place the link in `%userprofile%\Start Menu\Programs\Startup`.  Perhaps there's a more-appropriate environment variable that it could be using?  The link should go into the per-user folder (for your user account) whose contents are executed on login.  In your case, I'm guessing that might be `C:\Users\alistair\AppData\Roaming\Microsoft\Windows\Start Menu`?

**Comment by AlisterH** [comment]
Yes, start_pulseaudio.vbe was placed in ````C:\Users\alistair\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup````, so I put ````VcXsrv.lnk```` there as well.
The only environment variable I can see that might help is ````%APPDATA%````, which is set to ````C:\Users\alistair\AppData\Roaming````

**Comment by AlisterH** [comment]
FWIW ````C:\Users\alistair\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup```` is also where windows explorer opens if I do Start>Run>````shell:startup````
